packages/docs/src/routes/docs/(qwik)/getting-started/index.mdx

```diff
@@ -21,6 +21,7 @@ contributors:
   - kushalmahajan
   - sreeisalso
   - dustinsgoodman
+  - nsdonato
```

Comment:
I added myself because I had contributed to the styles section and forgot to add myself here 😅